LayoutTests/media/media-source/media-source-first-sample-pts-non-zero-canplay-without-renderer-expected.txt

```diff
@@ -0,0 +1,3 @@
+CONSOLE MESSAGE: This test verifies that the "canplay" event fires on a video element with a MediaSource that loads a video track whose first sample has a presentation timestamp greater than 0 when the video element is not in the DOM.
```

Comment:
the issue here is that per spec, we should stall that video.
That we allow a leeway near 0 is entirely user-agent dependent.

Reply:
That's fair, although Chrome and Firefox both dispatch "canplay" on this test case so we should too for compatibility. I'll look more closely at what the exact leeway we should implement in webkit.org/b/263413.